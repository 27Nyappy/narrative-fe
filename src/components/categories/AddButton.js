import React from 'react';
import PropTypes from 'prop-types';

const AddButton = ({ handleAddClick }) => {
  return (
    <button onClick={handleAddClick}>âŠ•</button>
  );
};

AddButton.propTypes = {
  handleAddClick: PropTypes.func.isRequired
};

export default AddButton;
