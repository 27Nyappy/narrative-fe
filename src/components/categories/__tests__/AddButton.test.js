import React from 'react';
import { shallow } from 'enzyme';
import AddButton from '../AddButton';

describe('AddButton component', () => {
  it('renders an add button', () => {
    const wrapper = shallow(<AddButton handleAddClick={() => true} />);
    expect(wrapper).toMatchSnapshot();
  });
});