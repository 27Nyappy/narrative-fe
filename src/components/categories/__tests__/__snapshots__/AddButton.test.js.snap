// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddButton component renders an add button 1`] = `
<button>
  âŠ•
</button>
`;
